packages/boilerplate-generator/boilerplate_web_template.js

```diff
@@ -0,0 +1,90 @@
+// Template function for rendering the boilerplate html
+// Replicates the template defined in boilerplate_web.browser.html
+// XXX Does not necessarily preserve formatting (e.g. additionalStaticJs newlines)
+// Arguments: root : { htmlAttributes, css : [{ url }], bundledJsCssUrlRewriteHook : Function, head, dynamicHead, body, dynamicBody, inlineScriptsAllowed, additionalStaticJs, meteorRuntimeConfig }
+
+Boilerplate_Web_Browser_Template = function(root) {
+  return [].concat(
+
+    // XXX is htmlAttributes ever anything but {}?
+    // may just be a generic Blaze/Spacebars thing.
+    function props1(htmlAttributes) {
+      return [
+        ['<html'].concat(Object.keys(htmlAttributes).map(function(key) {
+
+          // XXX probably need to wrap strings in "".
```

Comment:
Yeah, I've been delaying implementing this because I'm not sure what the best library for this kind of stuff. 

From what I can tell after reading https://github.com/meteor/blaze/blob/master/packages/htmljs/visitors.js#L198, https://html.spec.whatwg.org/multipage/syntax.html#attributes-2,

an attribute pair should stringify to `_.template(' <%= key %>="<%- value %>"')({key, value})`.

Potential issues: 
* The attribute namespace isn't explicitly handled and must be directly attached to the key of `htmlAttributes`, e.g. `htmlAttributes` may need to have entries like `{"xmlns:h": "http://www.w3.org/2000/xmlns/"}`.
* `_.template("<%- ... %>")` does HTML escaping, does not distinguish between *attribute mode*.  [this project](https://github.com/mathiasbynens/he) has an `isAttributeValue` option that may be useful. It seems to me that the attribute mode distinction isn't hugely important, and that most browsers handle unnecessarily escaped characters gracefully.